core-services/imagestream-importer/admin_rbac.yaml

```diff
@@ -19,7 +19,13 @@ items:
       - imagestreamimports
       verbs:
       - get
-      - update
+    - apiGroups:
+      - ""
```

Comment:
@cblecker This seems wrong

Reply:
This is how all the system roles are set up (both having, and lacking an apiGroup):
```
oc get clusterroles system:image-auditor -o yaml
apiVersion: authorization.openshift.io/v1
kind: ClusterRole
metadata:
  annotations:
    authorization.openshift.io/system-only: "true"
    openshift.io/reconcile-protect: "false"
  creationTimestamp: "2018-10-18T18:22:13Z"
  name: system:image-auditor
  resourceVersion: "128"
  selfLink: /apis/authorization.openshift.io/v1/clusterroles/system%3Aimage-auditor
  uid: bc646ed1-d302-11e8-a8ad-42010a8e0003
rules:
- apiGroups:
  - ""
  - image.openshift.io
  attributeRestrictions: null
  resources:
  - images
  verbs:
  - get
  - list
  - patch
  - update
  - watch
```

Reply:
okay, this is weird. Lets merge this, but I am interested in why its done this way.
/hold cancel